package.json

```diff
@@ -1343,12 +1343,6 @@
         "enablement": "calva:connected",
         "category": "Calva"
       },
-      {
-        "command": "calva.selectCurrentForm",
-        "title": "Select Current Form",
-        "category": "Calva",
-        "enablement": "editorLangId == clojure"
-      },
```

Comment:
I don't think we should retire command ids. They can be part of peoples keybindings configuration and Joyride scripts. Our options here is to either let it reside in the wrong category, or have two command ids point to the same function.

Reply:
Ah of course, I even discovered this missing command in the first place because I tried my custom `selectCurrentForm` shortcut with multicursor enabled and noticed it didn't do it! Thanks for catching this

Reply:
1. If we make a duplicate reference command:
   - 👍🏽 users can easily find it amongst `paredit.*` commands in the palette or the keyboard shortcuts UI or as part of autocomplete in the shortcuts JSON
   - 👍🏽 we can (maybe?) rename the existing command's title to "Select Current Form (legacy)" or something, to make it clearer as users discover commands that there's a "legacy" one and a "normal" one, even if they're really the same.
   - 👎🏽 this creates 2 duplicate commands with no obvious mechanism to migrate users' existing shortcuts (unless there _is_ a vsc system for that??)

2. If we keep the old, single command:
   - 👍🏽 simpler, no migration question even arises
   - 👎🏽 less obviously discoverable or grouped as part of `paredit`

Reply:
After thinking about it, I'm basically complete impartial

Reply:
wish we had analytics for command invocations haha

Reply:
i'll keep it at 1 cmd, but change the title